### PR TITLE
removed bitcoin support

### PR DESCRIPTION
**Pull request by maikelmclauflin**
rebased on current master, mostly just removing lines from codebase having to do with btc

### CODE REVIEW

**Comment by maikelmclauflin** [comment]
fixes #65

File: test/ledger.integration.test.js [hunk 1]
```diff
@@ -77,7 +77,6 @@ test('integration : v2 contribution workflow with uphold BAT wallet', async t =>
   t.true(response.body.hasOwnProperty('verification'))
 
   t.true(response.body.wallet.addresses.hasOwnProperty('BAT'))
-  t.true(response.body.wallet.addresses.hasOwnProperty('BTC'))
```
Comment (evq) [inline test/ledger.integration.test.js]: these are deposit addresses on a card denominated in BAT.
we still want users to be able to deposit BTC which will be autoconverted by uphold to BAT, so we should still address that this address is created

File: bat-utils/lib/runtime-wallet.js [hunk 2]
```diff
@@ -256,7 +228,6 @@ Wallet.providers.uphold = {
         }
         return { 'wallet': { 'addresses': {
           'BAT': ethAddr.id,
-          'BTC': btcAddr.id,
```
Comment (evq) [inline bat-utils/lib/runtime-wallet.js]: see below

File: ledger/controllers/registrar.js [hunk 3]
```diff
@@ -369,7 +342,6 @@ v2.createPersona =
       wallet: Joi.object().keys({
         paymentId: Joi.string().guid().required().description('opaque identifier for BTC address'),
         addresses: Joi.object().keys({
-          BTC: braveJoi.string().altcurrencyAddress('BTC').optional().description('BTC address'),
```
Comment (evq) [inline ledger/controllers/registrar.js]: see below

File: test/ledger.integration.test.js (hunk 1 above)
Comment (maikelmclauflin) [reply test/ledger.integration.test.js]: cool, i can put these marked lines back in.

**Pull request merged**